### DISCUSSION TITLE
老哥，貌似虎皮椒的支付接口还有点问题

### DISCUSSION THREAD

**Discussion by Ecalose** [Baiyuetribe/kamiFaka#46] [Bug反馈] [answered]
heroku版显示后端api接口没有工作，500错误

**Comment by Baiyuetribe** [comment, marked as answer]
我自己用的好好的，虎皮椒有两个API接口，请正确使用API接口，参考文档https://kmfaka.baklib-free.com/05a5/5d5b  下次麻烦仔细看下文档

**Reply by Ecalose** [reply]
我之前看过文档了，heroku上两个接口的确都显示后端接口未生效，我还新建测试了(T＿T)

**Reply by Baiyuetribe** [reply]
你找虎皮椒问API接口，如果不是文档里的两种，那就是三种API接口，第三个接口没人反馈已废弃

**Reply by Ecalose** [reply]
> 你找虎皮椒问API接口，如果不是文档里的两种，那就是三种API接口，第三个接口没人反馈已废弃

我问过虎皮椒的客服了，的确就是api.xunhupay.com这个接口啊，默认的那个是备用的，没有第三个接口了，然鹅heroku貌似还是不行

**Reply by Baiyuetribe** [reply]
我都说了，我自己用的虎皮椒接口，肯定是没问题的

**Reply by Ecalose** [reply]
> 我都说了，我自己用的虎皮椒接口，肯定是没问题的

heroku上是真的不行啊🤣，我还特地新建了一个啊，heroku上有的问题vps不一定有😂(就像之前那个当面付的bug一样），老哥可以试试我刚建的这个吗？麻烦老哥了
[https://zzzaa.herokuapp.com/](url)
后台都是默认的

**Reply by Baiyuetribe** [reply]
此BUG非彼BUG，我自用的难道还有问题？托你的请求，我亲自重新搭建并测试，接口依旧正常。请不要再这里继续反馈因你本人原因造成的故障。测试结果https://v16.herokuapp.com/
